library/src/main/java/com/github/javiersantos/piracychecker/activities/ActivityUtils.kt

```diff
@@ -4,15 +4,33 @@ import android.content.Context
 import android.os.Build
 import android.view.View
 
-fun Context.getAppName(): String = getString(applicationInfo.labelRes)
+internal fun Context.getAppName(): String {
+    var name: String = try {
+        (packageManager?.getApplicationLabel(applicationInfo) ?: "").toString()
```

Comment:
Perhaps it's better to use `.orEmpty()` instead of `?: ""`, what do you think?

Reply:
can't use `orEmpty` for `CharSequence`s